lib/ansible/modules/net_tools/haproxy.py

```diff
@@ -315,7 +336,7 @@ def get_state_for(self, pxname, svname):
         r = csv.DictReader(data.splitlines())
         state = tuple(
             map(
-                lambda d: {'status': d['status'], 'weight': d['weight']},
+                lambda d: {'status': d['status'], 'weight': d['weight'], 'scur': d['scur']},
```

Comment:
The number of active sessions for a backend server is given by the value of `scur` in [HTTP state checks](https://www.haproxy.com/doc/aloha/7.0/haproxy/healthchecks.html#checking-a-http-service).